api/src/main/java/org/pmiops/workbench/iam/IamServiceImpl.java

```diff
@@ -92,25 +82,21 @@ public List<String> revokeWorkflowRunnerRoleForUsers(
   /** Revokes life science runner role to list of service accounts. */
   private void revokeLifeScienceRunnerRole(
       String googleProject, List<String> petServiceAccountsLostAccess) {
-    com.google.api.services.cloudresourcemanager.v3.model.Policy policy =
-        cloudResourceManagerService.getIamPolicy(googleProject);
-    List<com.google.api.services.cloudresourcemanager.v3.model.Binding> bindingList =
-        Optional.ofNullable(policy.getBindings()).orElse(new ArrayList<>());
-    com.google.api.services.cloudresourcemanager.v3.model.Binding binding =
-        bindingList.stream()
-            .filter(b -> LIFESCIENCE_RUNNER_ROLE.equals(b.getRole()))
-            .findFirst()
-            .orElse(
```

Comment:
Creating a new Binding doesn't really make sense here.  The only reason we want to act on a Binding in this method is because it's one of those referenced in the Policy.